src/ILLink.Shared/TypeSystemProxy/WellKnownType.cs

```diff
@@ -28,10 +31,32 @@ public static (string Namespace, string Name) GetNamespaceAndName (this WellKnow
 				WellKnownType.System_Array => ("System", "Array"),
 				WellKnownType.System_Object => ("System", "Object"),
 				WellKnownType.System_Attribute => ("System", "Attribute"),
+				WellKnownType.System_NotSupportedException => ("System", "NotSupportedException"),
+				WellKnownType.System_Runtime_CompilerServices_DisablePrivateReflectionAttribute => ("System.Runtime.CompilerServices", "DisablePrivateReflectionAttribute"),
+				WellKnownType.System_Void => ("System", "Void"),
 				_ => throw new ArgumentException ($"{nameof (type)} is not a well-known type."),
 			};
 		}
 		public static string GetNamespace (this WellKnownType type) => GetNamespaceAndName (type).Namespace;
 		public static string GetName (this WellKnownType type) => GetNamespaceAndName (type).Name;
+		public static WellKnownType? GetWellKnownType (string @namespace, string name)
+		{
+			return @namespace switch {
+				"System" => name switch {
+					"String" => WellKnownType.System_String,
+					"Nullable`1" => WellKnownType.System_Nullable_T,
+					"Type" => WellKnownType.System_Type,
+					"Array" => WellKnownType.System_Array,
+					"Attribute" => WellKnownType.System_Attribute,
+					"Object" => WellKnownType.System_Object,
+					_ => null
```

Comment:
I think this is missing the new values you added:
- System_NotSupportedException 
- System_Runtime_CompilerServices_DisablePrivateReflectionAttribute 
- System_Void

Reply:
Nevermind, I saw that is not used by TryGetWellKnownType and only used by FindPredefinedType